src/GovUk.Education.ExploreEducationStatistics.Common/Services/FileStoragePathUtils.cs

```diff
@@ -8,54 +8,59 @@ public static class FileStoragePathUtils
     {
         public const string BatchesDir = "batches";
 
-        private static string PublicContentDownloadPath()
+        public static string PublicContentStagingPath()
         {
-            return "download";
+            return "staging";
```

Comment:
Does this not need `$"{AppendPathSeparator(prefix)}` adding to the start of it like the other paths?

Reply:
No because I don't intend that path to be prefixed by anything. It's the staging path that will be sometimes used to prefix the other paths, for example, 

`publications/pupil-absence-in-schools-in-england/releases/2016-17.json`

or with the staging path prefix:

`staging/publications/pupil-absence-in-schools-in-england/releases/2016-17.json`

Reply:
Cool ta